client-session/README.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: pattern
+title: Client Session Pattern
+folder: client-session
+permalink: /patterns/client-session/
+categories: Client Architecture
+language: en
+tags:
+  - Client distributed
+  - Stateless servers
+---
+
+## Name
+
+[Client Session pattern](https://dzone.com/articles/practical-php-patterns/practical-php-patterns-client)
+
+## Intent
+
+- Create stateless servers that removes the problem of clustering, as users can switch between servers seemlessly.
+- Makes data more resilient in case of server failover.
+- Works well with smaller data sizes.
+
+## Explanation
+
+Real-World Example
+
+> You're looking to create a data managment app allowing users to send requests to the server to modify and make changes to data stored on their devices. These requests are small in size and the data is individual to each user, negating the need for a large scale database implementation. Using the client session pattern, you are able to handle multiple concurrent requests, load balancing clients across different servers with ease due to servers remaining stateless. You also remove the need to store session IDs on the server side due to clients providing all the information that a server needs to perform their process.
+
+In Plain words
+
+> Instead of storing information about the current client and the information being accessed on the server, it is maintained client side only. Client has to send session data with each request to the server and has to send an updated state back to the client, which is stored on the clients machine. The server doesn't have to store the client information.([ref](https://dzone.com/articles/practical-php-patterns/practical-php-patterns-client))
+
```

Comment:
Please add at least the programmatic example here

Reply:
Added example.

Reply:
We need a bit more verbosity and description. Show the code for the used classes, describe their purposes and relations. The reader should be able to understand the design pattern by only reading this document.

Reply:
I have added additional description as requested. Please let me know if they are adequate.